app/src/main/res/values/colors.xml

```diff
@@ -30,4 +30,8 @@
     <color name="light_green">#0EBCA7</color>
     <color name="light_red">#E26767</color>
 
+    <color name="transaction_field_name">#BBC3C9</color>
```

Comment:
@danacealfonso 
Is this color used by all of the input fields?

Reply:
it is used in all my input fields label